README.md

```diff
@@ -26,7 +26,9 @@ following example script from `metriq-client/examples/metriq_hello.py`:
 ```python
 from metriq import MetriqClient
 
-client = MetriqClient(token="[Get this token from your web app account, and replace this string with it.]")
+#in your CLI, run the command: export METRIQ_CLIENT_API_KEY=<the key you get from the website>
```

Comment:
The `export ...` command will only work on Linux/Unix machines, so starting the comment with this would lead to the impression that this would also work on Windows. Also, I would suggest putting this information into a text cell above the Python snippet.

```suggestion
You will need to create an environment variable `` to store the Metriq API key. On Linux/Unix operating systems, you can run

export METRIQ_CLIENT_API_KEY=<the key you get from the website>

in your terminal. On Windows, you can similarly add this variable as an environment variable. 
```